nextcord/ui/select/string.py

```diff
@@ -348,3 +350,7 @@ def decorator(func: ItemCallbackType) -> ItemCallbackType:
         return func
 
     return decorator
+
+
+StringSelect = Select
```

Comment:
Do we need to alias this as `StringSelect`? Plain `Select` sounds fine to me.

Reply:
The alias is there because Discord renamed it, and we can be consistent with the naming when having StringSelect, UserSelect etc...